src/main/java/hudson/matrix/AxisDescriptor.java

```diff
@@ -80,7 +80,7 @@ public FormValidation doCheckName(@QueryParameter String value) {
      * necessary to disallow '=' in value as everything after the first
      * occurrence is considered to be a value.
      *
-     * Subclasses are expected to expose own <tt>doCheck</tt> method possibly delegating to this one.
+     * Subclasses are expected to expose own <code>doCheck</code> method possibly delegating to this one.
```

Comment:
```suggestion
     * Subclasses are expected to expose own {@code doCheck} method possibly delegating to this one.
```

is what I meant, but close enough.